tests/data/power_flow/automatic-tap-regulator/pgm-automatic-tap-line-drop-min/params.json

```diff
@@ -1,6 +1,7 @@
 {
   "calculation_method": "newton_raphson",
   "tap_changing_strategy": "min_voltage_tap",
+  "search_method": "binary_search",
```

Comment:
this is currently not exposed in the python API so it's not possible as of now to explicitly set this. it can cause a divergence between the C++ and Python validation tests

Reply:
Which is the reason current python batch validation tests fail (not directly). Will either remove this from json or add it to python api once we have a decision.